docs/UserGuide.md

```diff
@@ -175,18 +167,17 @@ _Details coming soon ..._
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous WorkBook home folder.
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Command summary
 
 Action | Format, Examples
 --------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
+**Add** | `add n/COMPANY_NAME d/YYYY-MM-DD e/EMAIL a/ADDRESS_OF_INTERNSHIP [t/TAG]…​` <br> e.g., `add n/Meta d/2022-09-15 e/hrmonkey@example.com a/John street, block 123, #01-01`
 **Clear** | `clear`
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List** | `list`
+**Edit** | `edit INDEX [n/COMPANY_NAME] [d/YYYY-MM-DD] [e/EMAIL] [a/ADDRESS_OF_INTERNSHIP] [t/TAG]…​​`<br> e.g.,`edit 2 n/META e/hrmonkey@example.com`
```

Comment:
Is `ADDRESS_OF_INTERNSHIP` even a prefix we want to support in WorkBook? Does not seem very relevant for the value prop/user stories we have. Open to discussions from everyone @tienyu2000 @adeearyaa @rylzxc

Reply:
Perhaps we could replace it with something like r/ROLE?